python/tskit/drawing.py

```diff
@@ -1285,16 +1423,18 @@ def __init__(
 
         subplot_x = self.plotbox.left
         container = self.get_plotbox()  # Top-level TS plotbox contains all trees
-        container["class"] = container["class"] + " trees"
+        container.set("class", container.get("class") + " trees")
         for subplot in subplots:
-            svg_subplot = container.add(
-                self.drawing.g(
-                    class_=subplot.svg_class,
-                    transform=f"translate({rnd(subplot_x)} {y})",
-                )
+            svg_subplot = ET.SubElement(
+                container,
+                "g",
+                {
+                    "class": subplot.svg_class,
+                    "transform": f"translate({rnd(subplot_x)} {y})",
+                },
             )
-            for svg_items in subplot.root_groups.values():
-                svg_subplot.add(svg_items)
+            for _, svg_items in subplot.root_groups.items():
```

Comment:
This seems a meaningless change. The original was better.

Reply:
Yeah, weird!